arbos/programs/params.go

```diff
@@ -170,6 +173,7 @@ func initStylusParams(sto *storage.Storage) {
 		ExpiryDays:       initialExpiryDays,
 		KeepaliveDays:    initialKeepaliveDays,
 		BlockCacheSize:   initialRecentCacheSize,
+		MaxWasmSize:      initialMaxWasmSize,
```

Comment:
This cannot be added here, as it retroactively changes stylus params for old arbos.

Param can be added to stylus in arbos40.
Before arbos 40 - you should not treat this param but use initialMaxWasmSize instead.